### DISCUSSION TITLE
I have successfully integrated rnpgp into Pass-Simple,

### DISCUSSION THREAD

**Discussion by shemeshg** [rnpgp/rnp#2169] [Show and tell]
[Pass Simple](https://sourceforge.net/projects/pass-simple/), a graphical user interface for passwordstore. Thanks to the excellent support from the rnp team, I was able to achieve this in just a few days. 
The rnpgp library enabled me to add features such as native Windows support, multithreading where possible, and cross-platform compatibility.

 I have tested the app on Linux, Windows, and Mac, and it works as expected. However, there are still some bugs that need to be fixed, 
 so installing the app on your machine might require some troubleshooting. To help you with that, I have written a guide that you can find here: https://shemeshg.github.io/pass-simple-mdbook/index.html

If you are interested in the source code of Pass Simple, you can check it out [here](https://github.com/shemeshg/pass-simple-qt) 

I hope you enjoy using Pass Simple and find it useful for managing your passwords securely.
I welcome any feedback or suggestions you might have. You can contact me through the project page or leave a comment below. 
 Thank you for your attention and support!

shemeshg

**Comment by ni4** [comment]
Thanks for sharing this!
